tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/css/fileDialog.css

```diff
@@ -13,7 +13,16 @@
 }
 
 .file-dialog-form-control {
-    width: 100%;
+    width: 83%;
```

Comment:
This is a common CSS for most of the dialogs, Create an additional one for export only